### PR TITLE
add 'late' field modifier

### PR DESCRIPTION
**Pull request by ivk1800**

### CODE REVIEW

**Comment by ivk1800** [comment]
@googlebot I signed it!

**Comment by ivk1800** [comment]
@googlebot I signed it!

**Comment by ivk1800** [comment]
maybe instead declare lated field add new type for FieldModifier: 'late' 'late final'?

**Comment by kevmoo** [comment]
Please update the changelog and bump the version

**Comment by kevmoo** [comment]
@natebosch @jakemac53 @matanlurey – LGTY?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 3.6.1
```
Comment (jakemac53) [inline CHANGELOG.md:1]:
```suggestion
## 3.7.0
```

The version also needs to change in the pubspec.

**Review by jakemac53 — COMMENTED** [review]
Variable declarations can also be late, we should add that at the same time?

File: CHANGELOG.md (hunk 1 above)
Comment (matanlurey) [reply CHANGELOG.md:1]: Yes, also `lated` is weird, can we either call it `late` (preferred) or `isLate`?

**Pull request merged**